### ISSUE TITLE
Badge has insufficient color contrast

### ISSUE THREAD

**Issue by stefanonardo** [ant-design/ant-design#49543]
### Reproduction link

[https://ant.design/~demos/badge-demo-basic](https://ant.design/~demos/badge-demo-basic)

### Steps to reproduce

Compute the ratio of background and foreground colors

### What is expected?

Color contrast should be greater than 4.5:1 following WCAG standards

### What is actually happening?

The color contrast is 3.26:1

| Environment | Info |
| --- | --- |
| antd | undefined |
| React | N/A |
| System | N/A |
| Browser | N/A |

**Comment by afc163** [comment]
You could customize `colorError` to a dark color: https://ant.design/docs/react/customize-theme

**Comment by stefanonardo** [comment]
Shouldn't the default value be already compliant to accessibility rules?

**Comment by afc163** [comment]
It is a complex job to upgrade the offcial color palettes, maybe next major version.

We will report this issue to our designer.
